projects/fw_102/src/fw_102.c

```diff
@@ -0,0 +1,38 @@
+#include <stdlib.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define COUNTER_INCREMENT_MS 500;  // 500 milliseconds between incrementing counter A
```

Comment:
I see that you've defined this up here but you haven't actually used it in your code. The whole point of this is so we don't have to hardcode 500. You can pass COUNTER_INCREMENT_MS in when you call your softtimers. The #define sort of acts like declaring a const variable. To understand why we use #define versus const variables, refer to this
https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/1591803936/C+Reference+for+Midnight+Sun+Firmware+Team

Reply:
I think I forgot to save it once I made that change to the rest of the code, my apologies. I updated it now.

Reply:
Oh yeah, almost forgot but there is a big mistake here. The #define statement doesn't need a semicolon at the end. Refer to firmware 102 to understand how to use this statement better.
https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/1282768912/Firmware+102+Supplementary+Material